linalg_ops/attention/generate_test_mlir_files.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# This script runs generate_e2e_attention_tests for all argument combinations that
+# we are interested in testing.
+#
+# The output is a 'generated' folder with contents like this:
+#   linalg_ops/
+#     convolution/
+#       generated/
+#         f16_f16_f16_f16/
+#           attention_f16_f16_f16_f16_large_calls.mlir
+#           attention_f16_f16_f16_f16_large.mlir
+#           attention_f16_f16_f16_f16_medium_calls.mlir
+#           attention_f16_f16_f16_f16_medium.mlir
+#           attention_f16_f16_f16_f16_small_calls.mlir
+#           attention_f16_f16_f16_f16_small.mlir
+# Usage:
+#   generate_test_mlir_files.sh
+
+set -euo pipefail
+
+this_dir="$(cd $(dirname $0) && pwd)"
+generated_dir_root="${this_dir}/generated"
+
+# Reset generated directory.
+rm -rf ${generated_dir_root?}
+mkdir -p ${generated_dir_root?}
+
+shapes=(
+  "small"
+  "medium"
+  "large"
+)
+
+# query_type;key_type;value_type;scale_type
+type_and_layout_combinations=(
+  "f16;f16;f16;f16"
+)
```

Comment:
The current test suite has not supported yet the other combinations. It will support; however, it needs some discussions - that is one of my upcoming tasks.